test-packages/integration-tests/test/cache.spec.ts

```diff
@@ -110,72 +110,33 @@ describe('CacheDestination & CacheClientCredentialToken', () => {
     delete process.env['VCAP_SERVICES'];
   });
 
-  it('caches the Destinations service call which fixed isolation Tenant (only simple auth flows)', async () => {
-    await populateDestinationsServiceCache();
+  it('caches the destinations service call with fixed isolation tenant (only simple auth flows)', async () => {
```

Comment:
I'm missing the context to understand why and how this test changed in this PR.

Reply:
This file is going to be deleted anyways (discussed with @deekshas8), as the tests in here don't really test anything and the cases are covered in other tests.